src/Compilers/CSharp/Portable/CommandLine/CSharpCompiler.cs

```diff
@@ -137,20 +137,28 @@ public override Compilation CreateCompilation(TextWriter consoleOutput, TouchedF
                 return null;
             }
 
-            var strongNameProvider = new LoggingStrongNameProvider(Arguments.KeyFileSearchPaths, touchedFilesLogger, _tempDirectory);
-
             var compilation = CSharpCompilation.Create(
                 Arguments.CompilationName,
                 trees.WhereNotNull(),
                 resolvedReferences,
                 Arguments.CompilationOptions.
                     WithMetadataReferenceResolver(referenceDirectiveResolver).
                     WithAssemblyIdentityComparer(assemblyIdentityComparer).
-                    WithStrongNameProvider(strongNameProvider).
                     WithXmlReferenceResolver(xmlFileResolver).
                     WithSourceReferenceResolver(sourceFileResolver));
 
-            return compilation;
+            var loggingOperations = new LoggingStrongNameFileSystem(touchedFilesLogger);
+            var searchPaths = Arguments.KeyFileSearchPaths;
+
+            bool fallback =
+                (compilation.Feature("BypassStrongName") != null) ||
+                (compilation.Options.CryptoKeyContainer != null);
```

Comment:
Are there any uses of `"BypassStrongName"`? If not, please remove. Same comment for VB.

Reply:
"BypassStrongName" is a command line switch.  Were tests in another branch of mine that were testing this, but I've just merged that branch in here now.